addons/mail/static/src/models/composer/composer.js

```diff
@@ -406,12 +406,20 @@ function factory(dependencies) {
                     ]],
                 });
             }
-            this.update({
-                [this.activeSuggestedRecordName]: [[
-                    'link',
-                    this[this.extraSuggestedRecordsListName][this[this.extraSuggestedRecordsListName].length - 1]
-                ]],
-            });
+            if (this[this.extraSuggestedRecordsListName][this[this.extraSuggestedRecordsListName]]) {
+                this.update({
+                    [this.activeSuggestedRecordName]: [[
+                        'link',
+                        this[this.extraSuggestedRecordsListName][this[this.extraSuggestedRecordsListName].length - 1]
```

Comment:
Would define some `const`(s) because it's hard to read.
_(the code of mention is hardly maintainable, I'll clean that when I find time, ideally after my `useModel` PR)_

Reply:
Following @seb-odoo comment, I don't think we need this `const` anymore. Whole PR is just a `return;` now.